packages/expect/src/__tests__/extend.test.ts

```diff
@@ -229,3 +230,18 @@ it('throws descriptive errors for invalid matchers', () => {
     'expect.extend: `default` is not a valid matcher. Must be a function, is "string"',
   );
 });
+
+it('exposes matchers in context', () => {
+  jestExpect.extend({
+    _shouldNotError(_actual: unknown, _expected: unknown) {
```

Comment:
Must be `shouldNotError`. It is declared at the top of this file, so using this name there will be no type error.

I reworked this just few hours ago ;D

Reply:
oh sorry, fixed it